src/chrome/chromeDebugAdapter.ts

```diff
@@ -314,7 +319,8 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
             }
         */
         telemetry.reportEvent('debugStarted', { request: 'attach', args: Object.keys(args) });
-        return this.doAttach(args.port, args.url, args.address, args.timeout, args.websocketUrl, args.extraCRDPChannelPort);
+        await this.doAttach(args.port, args.url, args.address, args.timeout, args.websocketUrl, args.extraCRDPChannelPort);
+        this.events.emitMilestoneReached(ChromeDebugSession.FinishedStartingUpEventName);
```

Comment:
I guess for launch this will be called from chrome-debug. For consistency, let's remove this and emit this from chrome-debug too.